### DISCUSSION TITLE
Highlighted text is unreadable in Rails guides when using dark mode

### DISCUSSION THREAD

**Discussion by Uaitt** [rails/rails#52606] [Docs]
### Steps to reproduce

On both the [7.2 guides](https://guides.rubyonrails.org/) and the [edge guides](https://edgeguides.rubyonrails.org/) highlighted text is very unreadable when dark mode is on.

![Screenshot 2024-08-14 at 16 57 07](https://github.com/user-attachments/assets/3126fe37-5217-4968-99ee-c0b9442b6ef5)

### Expected behavior

Text should be more readable when highlighted on the guides in dark mode.

### Actual behavior

The contrast is too low between the highlight background and text color.

**Comment by barnaclebarnes** [comment]
@Uaitt See https://github.com/rails/rails/pull/52593

It is fixed for me now (I reported a similar issue)

**Comment by Uaitt** [comment]
@barnaclebarnes I don't think this is a caching issue 🤔. I think the colours are correct, but their contrast is just too low.

**Comment by Uaitt** [comment]
This was fixed in #51499
